documentation/overview/account_setup/registration-and-login.md

```diff
@@ -3,89 +3,57 @@ ospool:
-will need to agree to our Acceptable Use Policy in order to get to the 
-main log in screen. 
+## Apply for OSPool Access
 
-The main log in screen will prompt you to sign in via your primary institutional 
-affiliation. You'll be directed to a discovery
-service which asks you what your home institution is. (If you've used CILogon
-before it may already know your home institution and skip this step.) Locate
-your institution in the list, or type its name to find matches. 
+To start, fill out the interest form on this OSG Portal site: 
 
-> **No Institutional Identity?**
-> 
-> If you don't have an institutional identity or can't find your institution 
-> on the provided list, either (separately) sign up for a Globus ID and follow 
-> the link for that option on this page, or contact the OSG Connect support 
-> team for guidance at support@osg-htc.org
+[OS Pool Account Request](https://portal.osg-htc.org/application)
 
-Note that this is the identity that will get linked to your OSG Connect account, 
-so be sure to pick the institution (if you have multiple affiliations) that 
-you would like to associate with your OSG Connect account. 
+This will send the Research Facilitation team an email. We will be in 
+touch to set up an orientation meeting, and confirm if you are joining 
+an existing project on the OSPool or starting a new one. 
 
-After selecting your institution in the discovery service, you'll be taken to
-your own institution's local sign-in screen. You've probably used it before,
-and if it looks familiar that's because it's exactly the same web site.  Sign in
-using your campus credentials. When done, you'll return automatically to the
-OSG Connect portal and can carry on with signup.
+## Orientation Meeting
 
-After continuing, and allowing certain permissions in the next screen, you'll be 
-asked to create a profile and save changes. If this works successfully, you should 
-see that your membership to OSG is "pending" on the right hand side of the screen. 
+The orientation meeting generally takes about 20-30 minutes and is a chance to 
+talk about your work, how it will 
+fit on the OSPool, and some practical next steps for getting started. 
 
-### Orientation Meeting
+## Register for an Access Point
 
-Once you've applied to join OSG Connect as described above, an OSG Connect support 
-team member will contact you to arrange an initial orientation meeting. This meeting 
-generally takes about 20-30 minutes and is a chance to talk about your work, how it will 
-fit on the OSG, and some practical next steps for getting started. Some of these 
-next steps are also listed below. 
+Before or during the orientation meeting, you will be prompted to register 
+for an account on a specific OSPool Access Point. The current default is the 
+"OSG Manage" Access Points, with registration via https://registry.cilogon.org. 
+You will be directed to follow instructions on [this page](../ap7-access) to register 
+for an account here. 
```

Comment:
```suggestion
"OSG Manage" Access Points.
```

Reply:
this registry link isn't the correct link to direct them to an enrollment flow - it's just the default comanage link.